src/configModel/configModel.vala

```diff
@@ -200,8 +200,47 @@ namespace SwayNotificationCenter {
         }
     }
 
+    public enum NotificationUrgencyEnum {
+        UNSET = -1,
+        LOW = 0,
+        NORMAL = 1,
+        CRITICAL = 2;
```

Comment:
Use the `UrgencyLevels` enums as values for these. Ex: `LOW = UrgencyLevels.LOW,`